recipes/default.rb

```diff
@@ -8,6 +8,7 @@
 
 # Make sure w32time service is running
 service 'w32time' do
+  run_as_user 'NT AUTHORITY\LocalService'
```

Comment:
For backward compatibility please protect that with a `if respond_to?(:run_as_user)`